syntax = "proto3";

package api;

option go_package = "./api";

message SearchReq {
  string search = 1;
}

message GetStatsResp {
  repeated StatsEntity statsEntities = 1;
}

message StatsEntity {
  int64 id = 1;
  string Username = 2;
  string FirstName = 3;
  string LastName = 4;
  int32 CockSize = 5;
}

service Stats {
  rpc Get(SearchReq) returns (GetStatsResp) {}
}
